lib/utils.js

```diff
@@ -1,66 +1,81 @@
 var basicAuth = require('basic-auth'),
-    config = require(__dirname + '/../app/config.js'),
-    fs = require('fs'),
-    portScanner = require('portscanner'),
-    prompt = require('prompt');
+  config = require(__dirname + '/../app/config.js'),
+  fs = require('fs'),
+  marked = require('marked'),
+  path = require('path'),
+  portScanner = require('portscanner'),
+  prompt = require('prompt'),
+  request = require('sync-request')
+
+// Variables
+var releaseUrl = null
 
 /**
```

Comment:
these comments belong to the basicAuth function, so they need to be moved back to being above it